client/components/gsuite/gsuite-new-user-list/style.scss

```
@@ -57,15 +42,51 @@
 	margin-top: 1.5em;
 }
 
+.gsuite-new-user-list__new-user .form-fieldset .form-label {
+	color: var( --color-text );
+	margin-bottom: 0;
+
+	.form-text-input-with-affixes__suffix {
+		font-weight: normal;
+	}
+
+	> div, > input {
+		margin-top: 5px;
+	}
+}
+
 .gsuite-new-user-list__new-user-section {
 	@include breakpoint-deprecated( '>800px' ) {
 		display: flex;
 		justify-content: space-between;
 
-		& > *:not(:first-child) {
+		& > *:not( :first-child ) {
 			margin-left: 15px;
 		}
 	}
+
+	.gsuite-new-user-list__new-user-remove-user-button {
+		display: none; // TODO: Fix button displayed in the middle of form (see 1165571745647867-as-1198174007918339)
+		overflow: unset; // Prevents ellipsis from appearing because of a right margin applied on the gridicon
+
+		@include breakpoint-deprecated( '>800px' ) {
+			display: block;
+			height: 40px;
+			width: 50px;
+
+			span {
+				display: none;
+			}
+		}
+	}
+
+	&.show-labels {
+		.gsuite-new-user-list__new-user-remove-user-button {
+			@include breakpoint-deprecated( '>800px' ) {
+				margin-top: calc( 1.5em + 5px );
+			}
+		}
+	}
```

Could we maybe make this more readable and possibly save on the braces if we reduced the nesting?

```suggestion
	&.show-labels  .gsuite-new-user-list__new-user-remove-user-button {
			@include breakpoint-deprecated( '>800px' ) {
				margin-top: calc( 1.5em + 5px );
			}
	}
```

---

I took care of this partly by changing the element that `show-labels` is attached to, partly by moving this into the same section as the label CSS, and by shifting the CSS to one less nesting level.